app/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt

```diff
@@ -92,7 +92,13 @@ interface Pixel {
         WIDGET_LEGACY_CTA_DISMISSED("m_wlc_d"),
         WIDGETS_ADDED(pixelName = "m_w_a"),
         WIDGETS_DELETED(pixelName = "m_w_d"),
-        WIDGET_LAUNCHED(pixelName = "m_w_l"),
+
+        APP_WIDGET_LAUNCH(pixelName = "m_w_l"),
+        APP_ASSIST_LAUNCH(pixelName = "m_a_l"),
+        APP_GOOGLE_BAR_LAUNCH(pixelName = "m_g_l"),
```

Comment:
`GOOGLE_BAR` is maybe not the best name for this. It's only "Google" if the user selected Google; could we call it something like `SYSTEM_SEARCH_WIDGET` instead? (open to suggestions on a better name!)

Reply:
How about SYSTEM_SEARCH_BOX?